include_testcases/ideal_MHD_far_dipole.h

```diff
@@ -0,0 +1,104 @@
+// This project is licensed under the terms of the Creative Commons CC BY-NC-ND 4.0 license.
+
+//--------------------------------------------------------------------------------------------------
+// Functions for field initialisation
+//--------------------------------------------------------------------------------------------------
+
+
+// Example for inhomogenious magnetic fields
+// A dipole outside of the domain
+// paramters
+REAL4 constant M = {0.0, 0.0, 1.0, 0.0};
+REAL4 constant X_0 = {-1.0, 0.5, 0.0, 0.0};
+REAL constant A0 = 0.1;
+REAL constant omega = 6.28 * 10;
+/*
+There is no analytical solution for this simulation
+*/
+inline REAL4 u_analytical(uint ix, uint iy, uint iz, REAL time) {
+
+	return (REAL4) {(REAL)(0), (REAL)(0), (REAL)(0), (REAL)(0)};
+}
+
+inline REAL4 b_analytical(uint ix, uint iy, uint iz, REAL time) {
+
+	return (REAL4) {(REAL)(0), (REAL)(0), (REAL)(0), (REAL)(0)};
+}
+
+// Magnetic Dipole definition out of 
+// Course of Theoretical Physics: The Classical Theory of Fields
+// L.D. Landau and E. M. Lifschitz
+// (1951)
+
+inline REAL4 b_dipole(REAL4 X, REAL4 M) {
+	REAL inorm = 1 / sqrt(X.x*X.x + X.y*X.y + X.z*X.z);
```

Comment:
Using `rsqrt(...)` instead of `1 / sqrt(...)` should be faster and more accurate.